config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
+  get 'users/:id', to: 'users#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:id', to: 'posts#show'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  root 'users#index'
```

Comment:
- [ ] **[OPTIONAL]** I think we can improve the code here by using the alternative `resources` helper as provided by Rails.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
True 💡 handling that right now, thanks @TedLivist 🙂